/**:
  ros__parameters:
    onnx_model_path: ""
    args_path: ""
    planning_frequency_hz: 10.0
    debug_params:
      publish_debug_route: true
      publish_debug_map: false
